lib/ui/lobby.js

```diff
@@ -0,0 +1,87 @@
+import { el, list } from 'redom';
+
+import { dispatch } from './dispatch';
+
+
+class GameInfo {
+  constructor() {
+    this.el = el('tr',
+      this.gameModeDisplay = el('td'),
+      this.playerCountDisplay = el('td')
+    );
+
+    this.el.onclick = ev => {
```

Comment:
It certainly is not perfect, but I do find multi-line arrow functions more readable with the parenthesis.